JustBurp/myapp/views/partials/chefDailyTimeSetupCard.hbs

```diff
@@ -7,58 +7,99 @@
 
 <script>
     $(function () {
-        $('.time').timepicker();
-    });
+        $('.prepareStartTime').timepicker({
+            disableTextInput: true,
+            'maxTime': '11:00pm'
+        });
+
+        $('.startTime').timepicker({
+            disableTextInput: true,
+            'minTime': '5:00pm',
+            'maxTime': '11:00pm'
+        });
+        $('.endTime').timepicker({
+            disableTextInput: true,
+            'minTime': '5:00pm',
+            'maxTime': '11:00pm'
+        });
```

Comment:
I think Dj forget to told you that the chef have to commit at least one hour cooking time if he or she decides to cook at this day. In this case, chef may not set up the startTime later than 10:00pm and endTime earlier than 6pm.